tfhe/docs/guides/run_on_gpu.md

```diff
@@ -218,70 +218,3 @@ Depending on the platform, this can restrict the number of GPUs used to perform
 There is **nothing to change in the code to execute on multiple GPUs**, when 
 they are available and have peer access to GPU 0 via NVLink. To keep the API as user-friendly as possible, the configuration is automatically set, i.e., the user has no fine-grained control over the number of GPUs to be used.
 
```

Comment:
Here I think we can add a sentence to link to the benchmarks page. :slightly_smiling_face:

Reply:
Great idea, I added the reference for zk proof one also.